pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "ipyleaflet",
     "ipycytoscape",
     "tabulate",
-    "fabrictestbed==1.5.4",
+    "fabrictestbed==1.5.5",
```

Comment:
Can we change this 1.5.6 please? I missed a credmgr dependency which was in another PR and hadn't been pushed.

Reply:
one sec